server/bert_serving/server/bert/extract_features.py

```diff
@@ -91,9 +91,14 @@ def convert_lst_to_features(lst_str, max_seq_length, max_position_embeddings,
         # For classification tasks, the first vector (corresponding to [CLS]) is
         # used as as the "sentence vector". Note that this only makes sense because
         # the entire model is fine-tuned.
-        tokens = ['[CLS]'] + tokens_a + ['[SEP]']
+        if special_token is False and is_tokenized and not tokens_b:
```

Comment:
change it to `if no_special_token and is_tokenized and not tokens_b:`

Reply:
you are right, that's sound much better